com.unity.render-pipelines.high-definition/Runtime/Material/Decal/ClearPropertyMaskBuffer.compute

```diff
@@ -5,14 +5,20 @@
 
 #pragma kernel CSMain
 
+#ifdef PLATFORM_SUPPORTS_BUFFER_ATOMICS_IN_PIXEL_SHADER
```

Comment:
Question: do we support any platform that doesn't have this defined? This is defined for D3D11/12, Vulkan and I assume on Console it is too.

Reply:
wasn't on Metal at some point, so for now I keep it :)

Reply:
It is defined on it now, but sure doesn't hurt.